package.json

```diff
@@ -83,6 +83,7 @@
   "dependencies": {
     "@okta/okta-auth-js": "1.8.0",
     "backbone": "1.2.1",
+    "clipboard": "1.6.1",
```

Comment:
It is - the one downside to this approach (over using something like [lerna](http://lernajs.io/) which is the direction we should move towards) is that by checking in these dependencies, we have no way of alerting npm about *their* dependencies. In this case, clipboard is the one dependency we need to pull in.

Btw, this is going to be more of an issue as we're currently moving all our courage dependencies out of source and into package.json.